packages/utils/src/gasEstimates.ts

```diff
@@ -1,6 +1,8 @@
 export const GAS_ESTIMATES = {
```

Comment:
see previous comments

Reply:
pending... Need to fetch this add to gasEstimates @rhlsthrm